### DISCUSSION TITLE
I can't download

### DISCUSSION THREAD

**Discussion by ibrahim474** [aclap-dev/video-downloadhelper#319] [Help and Q&A]
### Your question

Hello, although I tried many times, I could only download 2 videos.

The general errors I get are as follows.

Exit code: 1
Error: Exit code: 1

    at as.receive (chrome-extension://pfoiagbblcbmognbkekfpodpidedkmcc/background/main.js:1:13157)
    at chrome-extension://pfoiagbblcbmognbkekfpodpidedkmcc/background/main.js:1:42622

SideDownload error:
[hls @ 0x5973d80] Error when loading first segment 'https://vz-1e0d00d9-cfc.bnycdn.net/9dc1f4b3-67ea-45f0-a7c4-cdec8fc718c2/1280x720/video0.ts?v=0&resolution=1280x720&server=1366 &contextId=8d87ece9 -0be5-43da-8fe4-fe29d64d8cc2'
[in#0 @ 0x5973c80] Error opening input: Invalid data found when processing input
Error opening input file https://iframe.mediadelivery.net/9dc1f4b3-67ea-45f0-a7c4-cdec8fc718c2/1280x720/video.drm?contextId=8d87ece9-0be5-43da-8fe4-fe29d64d8cc2.
Error opening input files: Invalid data found when processing input

### Addon & CoApp version and details

version: 8.2.2.9
channel: beta
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: tr
license: unneeded
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "/home/ibrahim/.local/share/vdhcoapp/vdhcoapp",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "linux",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "/home/ibrahim",
  "converterBinary": "/home/ibrahim/.local/share/vdhcoapp/ffmpeg",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360"
}

**Comment by Mattmjs** [comment]
This matches another case : https://github.com/aclap-dev/video-downloadhelper/discussions/29
Read the marked as answer.

**Reply by ibrahim474** [reply]
Unfortunately I'm getting the same error. I removed the ignore tick for low quality videos, but the problem persists.

**Reply by Mattmjs** [reply]
What does it show in the drop down menu if you post a screenshot.

**Reply by ibrahim474** [reply]
![Screenshot_20240322_153439](https://github.com/aclap-dev/video-downloadhelper/assets/41086890/0f657b82-33a8-48b7-b61d-5ca79cbfac60)
![Screenshot_20240322_153329](https://github.com/aclap-dev/video-downloadhelper/assets/41086890/7c97fe82-29c7-4e34-aeef-641289ccc929)
![Screenshot_20240322_153254](https://github.com/aclap-dev/video-downloadhelper/assets/41086890/1060e77d-8210-481b-98b3-4c4fda0a57f7)

**Reply by Mattmjs** [reply]
Ok looks like it only offers one HLS mp4 to download even after changing the low quality setting. It doesn't look like it's going to be
possible. Unless @paulrouget has anything further to add.
